.ci/build.sh

```diff
@@ -48,3 +50,5 @@ python manage.py dumpdata meta_review > _site/$META_REVIEW_DATA
 
 python manage.py collectstatic --noinput
 python manage.py distill-local public --force
+
+rm db.sqlite3
```

Comment:
add this to your `rm -rf` above.

this isnt isnt about removing artifacts after the build.

we want to keep these artifacts, so someone can debug the database if the built site looks strange.

but if they re-run the build, the database needs to be empty again.